### PR TITLE
Section on PI's red LED in Troubleshooting

### PR DESCRIPTION
**Pull request by marc-tonsen**

### CODE REVIEW

File: src/invisible/user-guide/troubleshooting.md [hunk 1]
```diff
@@ -19,3 +19,12 @@ description: Need assistance? Check out this section for troubleshooting with Pu
    cable rather than another 3rd party cable. Such cables are available
    in a wide variety of quality and using cables of insufficient quality
    can lead to loss of connection.
+
+**A red LED is blinking on my Pupil Invisible glasses!**
+
+The LED is indicating that there is a problem that may affect further data collection and tries to grab your attention. To get details on what the problem exactly is open the Pupil Invisible Companion app, which will show an error description.
```
Comment (papr) [inline src/invisible/user-guide/troubleshooting.md:25]:
```suggestion
The LED is indicating that there is a problem that may affect further data collection and tries to grab your attention. To get details on what the problem exactly is, open the Pupil Invisible Companion app, which will show an error description.
```

File: src/invisible/user-guide/troubleshooting.md [hunk 2]
```diff
@@ -19,3 +19,12 @@ description: Need assistance? Check out this section for troubleshooting with Pu
    cable rather than another 3rd party cable. Such cables are available
    in a wide variety of quality and using cables of insufficient quality
    can lead to loss of connection.
+
+**A red LED is blinking on my Pupil Invisible glasses!**
+
+The LED is indicating that there is a problem that may affect further data collection and tries to grab your attention. To get details on the problem, open the Pupil Invisible Companion app, which will show an error description.
```
Comment (willpatera) [inline src/invisible/user-guide/troubleshooting.md:25]:
```suggestion
The LED blinks to try to get the wearer's attention. The red blinking LED indicates that there is a problem that may affect further recording. To get details, open the Pupil Invisible Companion app to view an error description.
```

File: src/invisible/user-guide/troubleshooting.md [hunk 3]
```diff
@@ -19,3 +19,12 @@ description: Need assistance? Check out this section for troubleshooting with Pu
...
+
+The potential problems incude:
```
Comment (willpatera) [inline src/invisible/user-guide/troubleshooting.md:27]:
```suggestion
The potential problems include:
```

File: src/invisible/user-guide/troubleshooting.md [hunk 4]
```diff
@@ -19,3 +19,12 @@ description: Need assistance? Check out this section for troubleshooting with Pu
...
+- Low battery on the Companion device.
```
Comment (willpatera) [inline src/invisible/user-guide/troubleshooting.md:28]:
```suggestion
- Low battery on the Companion Device.
```

File: src/invisible/user-guide/troubleshooting.md [hunk 5]
```diff
@@ -19,3 +19,12 @@ description: Need assistance? Check out this section for troubleshooting with Pu
...
+- Little storage left on the Companion device.
```
Comment (willpatera) [inline src/invisible/user-guide/troubleshooting.md:29]:
```suggestion
- Low storage space remaining on the Companion Device.
```

File: src/invisible/user-guide/troubleshooting.md [hunk 6]
```diff
@@ -19,3 +19,12 @@ description: Need assistance? Check out this section for troubleshooting with Pu
...
+- Unstable connection to the Pupil Invisible device during a recording.
```
Comment (willpatera) [inline src/invisible/user-guide/troubleshooting.md:30]:
```suggestion
- Unstable connection to the Pupil Invisible Glasses device during a recording.
```

**Review by willpatera — CHANGES_REQUESTED** [review]
@marc-tonsen I made quite a few changes. Please feel free to accept/edit my suggestions.

**Comment by marc-tonsen** [comment]
Thanks for the edits @willpatera, I accepted all of them. Ready to merge I think!

**Pull request merged**